### PR TITLE
CT-3027 non offender sars case closure in calendar days

### PR DESCRIPTION
**Pull request by njpreddy**
## Description

CT 3027 Case closure time taken should be calculated in calendar days for non-offender SARs, matching offender SARs behaviour.

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
* [x] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`

### Screenshots

Before:
Before

After:
After

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=CT&modal=detail&selectedIssue=CT-3027

### Manual testing instructions

Create a closed offender and non-offender SAR.  In both cases, depending on the difference between received and responded dates, the 'Time taken' will be calculated in calendar days not working days.

### CODE REVIEW

File: app/decorators/case/sar/offender_decorator.rb [hunk 1]
```diff
@@ -3,31 +3,29 @@ class Case::SAR::OffenderDecorator < Case::BaseDecorator
...
 
-  def back_link(mode, previous_step)
-    url = if mode == :edit
-            h.case_path(id)
-          else
-            "#{h.step_case_sar_offender_index_path}/#{previous_step}"
-          end
-    h.link_to "Back", url, class: 'govuk-back-link'
+  def third_party_display
+    object.third_party == true ? 'Yes' : 'No'
```
Comment (willmcb) [inline app/decorators/case/sar/offender_decorator.rb:25]: I think here we should just be able to do:

`object.third_party` ? 'Yes' : 'No'

As I think `third_party` its self is either `true` or `false`

**Review by willmcb — APPROVED** [review]
Good job - a few small non-blocking points to consider 👍

File: app/decorators/case/sar/offender_decorator.rb (hunk 1 above)
Comment (njpreddy) [reply app/decorators/case/sar/offender_decorator.rb]: Done! :-)

File: spec/support/features/steps/cases/responding_steps.rb [hunk 2]
```diff
@@ -73,39 +58,37 @@ def close_sar_case_step(timeliness: 'in time', tmm: false, editable: true)
...
-  expect(show_page.response_details.timeliness.data.text)
-    .to eq "Answered #{timeliness}"
+  expect(show_page.response_details.date_responded.data.text).to eq Date.today.strftime(Settings.default_date_format)
+  expect(show_page.response_details.timeliness.data.text).to eq "Answered #{timeliness}"
 
   # Regex required to handle both cases of 1 or more days to respond
-  expect(show_page.response_details.time_taken.data.text).to match(/\d+ working day[s{1}]?/)
+
+  if show_page.has_css?('.overturned-sar-basic-details')
+    expect(show_page.response_details.time_taken.data.text).to match(/\d+ working day[s{1}]?/)
```
[the remark this answers is not in the document]
Comment (njpreddy) [reply spec/support/features/steps/cases/responding_steps.rb:75]: Done! :-)

**Pull request merged**